registrations/models.py

```diff
@@ -251,7 +251,40 @@ def fire_state_metric(sender, instance, created, **kwargs):
 
             total = get_or_incr_cache(
                 total_key,
-                registrations_for_state(state).count)
+                registrations_for_identity_field(
+                    {"details__state": state}).count)
```

Comment:
it returns a reference to the `count()` function
`get_or_incr_cache` takes the key to check and a method to call if the count isn't in the cache